calendarScript.js

```diff
@@ -0,0 +1,74 @@
+const date = new Date();
+
+const renderCalendar = () => {
+
+    date.setDate(1);
+
+    const monthDays = document.querySelector(".days");
+
+    const lastDay = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDate();  // 0 = last day of pre month, 1 = first day of current month //
+
+    const prevLastDay = new Date(date.getFullYear(),
+    date.getMonth(), 0).getDate();
+
+    const firstDayIndex = date.getDay();
+    const lastDayIndex = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDay();
+
+    const nextDays = 7 - lastDayIndex - 1;
+
+
+    const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December"
+    ];
+
+    document.querySelector(".date h1").innerHTML = months[date.getMonth()];
+
+    document.querySelector(".date p").innerHTML = new Date().toDateString();
+
+    let days ="";
+
+    for(let x = firstDayIndex; x >0; x--) {
+        days += `<div class = "prev-date">${prevLastDay - x + 1}</div>`
+    }
+
+
+    for(let i = 1; i <= lastDay; i++) {
+        if(i === new Date().getDate() && date.getMonth() === new Date().getMonth())
+            {days += `<div>${i}</div>`; }
+        else {
+            days += `<div>${i}</div>`;
+        }
+    }
+
+    for(let i = 1; i <= nextDays; i++) {
+        days += `<div class = "next-date">${i}</div>`;
+        monthDays.innerHTML = days;
+    }
+
+}
+
+
+document.querySelector('.prev').addEventListener('click', () => {
```

Comment:
```suggestion
document.querySelector('.prev').addEventListener("click", () => {
```

전체적으로 double quote를 사용하셨는데, 이 부분만 single quote로 되어있습니다🤔
JS에서는 둘 다 사용되기 때문에, 어떤 것을 사용하셔도 문제는 없습니다만 프로젝트 전반적으로는 일관성을 지켜주시는 것이 좋습니다.